akvo/rsr/views/py_reports/kickstart_word_report.py

```diff
@@ -81,14 +81,13 @@ def build_log_frame(project_view):
                     previous_indicator = indicator.title
                     current_indicator = indicator.title
                 disaggregations = []
-                for d in period.disaggregations.all():
+                for d in period.disaggregations.values():
                     disaggregations.append({
-                        'label': str(d.dimension_value),
-                        'value': d.value
+                        'label': str(d['type']),
+                        'value': d['value']
```

Comment:
I think it would be good to use defensive programming here and use `.get`

Reply:
Fixed